helm-frontend/src/components/RowValue.tsx

```diff
@@ -41,7 +42,10 @@ export default function RowValue({ value, title }: Props) {
     if (href) {
       return (
         <Link to={href} inTable title={title}>
-          {formatNumber(value.value)}
+          <div className="flex items-center">
+            {formatNumber(value.value)}
+            <ArrowTopRightOnSquareIcon className="w-3 h-3 ml-1 opacity-40" />
```

Comment:
Consider reducing opacity

Reply:
Will reduce it further slightly before merging